code/src/App.css

```diff
@@ -0,0 +1,449 @@
+} 
+
+.fill-in-field {
+background-color:#D1CCC1;
+color: #824947;
+font-family: 'Monaco';
+height: 30px;
+width: 150px;
+font-size: 20px;
+border: solid 1px #824947;
+}
+
+/* BUTTONS */
+
+.button-container {
+display: flex;
+flex-direction: column;
+align-items: center;
+justify-content: center;
+}   
+
+.p-button-box {
+  font-family: 'Poppins', sans-serif;
+  color: #37372C;
+  font-weight:bold;
+  font-size: 20px;
+}
+
+.button-box {
+  display:block;
+  position: relative; 
+  padding-left: 35px;
+  margin-bottom: 12px;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
+  user-select: none;
+  cursor: pointer;
+  font-family: 'Monaco';
+  color: #824947;
+  font-weight:bold;
+}
+
+.button-small-box {
+  width: 80%;
+}
+
+.button-box input {
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+  
+}
+
+.checkmark {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: 20px;
+  width: 20px;
+  background-color: #AD8841;
+  border-radius: 50%;
+}
+
+.button-box input:checked ~ .checkmark {
+  background-color: #824947;
+}
+
+.checkmark:after {
+  content: "";
+  position: absolute;
+}
+
+.button-box input:checked ~ .checkmark:after {
+  display: block;
+}
+
+.button-box .checkmark:after {
+  top: 6px;
+ left: 6px;
+ width: 8px;
+ height: 8px;
+ border-radius: 50%;
+ background: #D1CCC1;
+}
+
+
+/* DROP-DOWN */
+
+.drop-down-box {
+display: flex;
+flex-direction: column;
+align-items: center;
+width: 80%;
+margin: auto;
+padding-top: 20px;
+padding-bottom: 20px;
+}
+
+.p-drop-down-box {
+  font-family: 'Poppins', sans-serif;
+  color: #37372C;
+  font-weight:bold;
+  text-align: center;
+}
+
+.pick-favorite {
+  font-family: 'Poppins', sans-serif;
+  color: #824947;
+  font-weight: bold;
+  padding: 3px;
+}
+
+.pick-field {
+  background-color:#D1CCC1;
+  color: #824947;
+  font-family: 'Monaco';
+  height: 30px;
+}
+.opt-po {
+  background-color:#D1CCC1;
+  color: #824947;
+  font-family: 'Monaco';
+  font-weight: bold;
+}
+
+
+/* SUBMIT */
+
+.sub-button-box {
+  display: flex;
+  justify-content: center;
+  padding: 20px 0 20px 0;
+
+}
+
+.submit-button {
+background-color: #D1CCC1;
+border: solid 1px #824947;
+height: 30px;
+font-family: 'Monaco';
+color: #37372C;
+font-size: 20px;
+}
+
+
+/* RESULT */
+
+.result-box {
+display:flex;
+flex-direction: column;
+align-items: center;
+width: 80%;
+margin: auto;
+}
+
+.p-result-box {
+  font-family: 'Poppins', sans-serif;
+  color: #824947;
+  font-weight: bold;
+  font-size: 25px;
+}
+
+
+.p-result-box-sum {
+  font-family: 'Monaco';
+  color: #37372C;
+  text-align: center;
+  font-size:20px;
+  font-weight: bold;
+}
+
+.ice-pic {
+  width: 300px;
+  height: 300px;
+  border: solid 1px #824947;
+  border-radius: 50%;
+  margin: 5px;
+}
+
+.again-btn-box {
+  display: flex;
+}
+
+.again-btn {
+  margin: 10px;
+  font-family: 'Monaco';
+  background-color: #AD8841;
+  color: #37372C;
+  border: solid 1px #824947;
+  height: 30px;
```

Comment:
remember that you can set css declarations for multiple classes if they have the same styling(or in part the same styling) so you can save some space! For example:
.again-btn,
.ice-pic {
   border:
   }
and then make the individual declarations on respective. Now it was only one in common in this example hehe but there are some classes above with more declarations in common !